packages/pn-pa-webapp/src/components/Notifications/ConfirmCancellationDialog.tsx

```diff
@@ -36,11 +34,9 @@ const ConfirmCancellationDialog: React.FC<Props> = ({ showModal, onClose, onConf
   }, [showModal]);
 
   return (
-    <Dialog open={showModal} data-testid="cancel-notification-modal">
-      <DialogTitle id="dialog-title" sx={{ p: 4 }}>
-        {t('detail.cancel-notification-modal.title')}
-      </DialogTitle>
-      <DialogContent sx={{ px: 4, pb: 4 }}>
+    <PnDialog open={showModal} data-testid="cancel-notification-modal">
```

Comment:
Checks the comments left on CodeModal about distance between Title and Content, and Content and Actions